incountry/crypto_utils.py

```diff
@@ -103,12 +104,20 @@ def hash_object_record_keys(obj, salt, normalize_keys=False, hash_search_keys=Tr
     return res
 
 
-def encrypt_record(crypto, record, salt, normalize_keys=False, hash_search_keys=True):
+def encrypt_record(
+    crypto: InCrypto,
+    record: dict,
+    salt: str,
+    keys_to_hash: list = KEYS_TO_HASH,
+    keys_to_encrypt: list = EXTRA_KEYS_TO_ENCRYPT,
+    normalize_keys: bool = False,
+    hash_search_keys: bool = True,
+):
```

Comment:
Return types could be really useful for us as well:

```suggestion
)  -> Dict[str, Any]:
```